src/Processors/QtiV2/Out/Validation/AbstractQuestionValidationBuilder.php

```diff
@@ -12,7 +13,7 @@ abstract class AbstractQuestionValidationBuilder
 
     abstract protected function buildResponseDeclaration($responseIdentifier, $validation);
 
-    public function buildValidation($responseIdentifier, $validation, $isCaseSensitive = true)
+    public function buildValidation($responseIdentifier, $validation, $distractorRationaleResponseLevel = array(), $isCaseSensitive = true)
```

Comment:
Sorry I didn't pick this up sooner - you've added a new optional parameter `$distractorRationaleResponseLevel` before the existing parameter `$isCaseSensitive`, which breaks the order of arguments. This will break calling classes that already pass `$isCaseSensitive`, unless we refactor the calls to match. See the `shorttext` question type here: https://github.com/Learnosity/learnosity-qti/blob/develop/src/Processors/QtiV2/Out/QuestionTypes/ShorttextMapper.php#L42

Be careful of making breaking changes like this - you'll either have to find and refactor all method calls, or add the new parameters at the end.

Reply:
Done